experimental/auth-logic/wrapper_interface_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package authlogic contains logic and tests for interfacing with the
+// authorization logic compiler
+package authlogic
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
+
+type IntPair struct {
```

Comment:
```suggestion
type intPair struct {
```

Or do you actually need to export this name?

Reply:
exporting not needed, thanks. fixed